dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -139,15 +140,16 @@ public fun DataFrame.Companion.readSqlTable(
     val url = connection.metaData.url
     val dbType = extractDBTypeFromConnection(connection)
 
-    val selectAllQuery = if (limit > 0) dbType.sqlQueryLimit("SELECT * FROM $tableName", limit)
-    else "SELECT * FROM $tableName"
+    val selectAllQuery = if (limit > 0) {
+        dbType.sqlQueryLimit("SELECT * FROM $tableName", limit)
```

Comment:
and again, here we don't need these brackets in the code

Reply:
It's more readable for multiline if-statements: https://pinterest.github.io/ktlint/latest/rules/standard/#if-else-bracing and https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements.
No brackets in if-statements are only allowed/clear when it fits on the same line.